### ISSUE TITLE
DietPi-Software | Emby server: Install fails on ARM+Jessie due to too old libc6

### ISSUE THREAD

**Issue by MichaIng** [MichaIng/DietPi#2521]
Ref: https://dietpi.com/phpbb/viewtopic.php?f=11&t=5502
🈴 Check libc6 backports: No backports available!
🈴 Check libc6 install manually from e.g. Stretch: https://github.com/Fourdee/DietPi/issues/2521#issuecomment-462053775
~- [ ] Force to offer Odroid C1 Stretch image 😉~ **EDIT: Related new issue: https://github.com/Fourdee/DietPi/issues/2561**
- [x] Build own ARM Jessie package or disable for ARM + Jessie

**Comment by MichaIng** [comment]
Okay couldn't replicate on Jessie VM 🤔:
```
 DietPi-Software
─────────────────────────────────────────────────────
 Mode: Installing Emby Server: web interface media streaming server

[  OK  ] DietPi-Software | Connection test: https://github.com/MediaBrowser
[  OK  ] DietPi-Software | wget https://github.com/MediaBrowser/Emby.Releases/releases/download/3.5.2.0/emby-server-deb_3.5.2.0_amd64.deb -O 41.deb
[  OK  ] DietPi-Software | Installing deb package
[  OK  ] DietPi-Software | Cleaning download directory
```

So it's arm/armhf/Odroid C1 specific.
____________
Rechecked dependencies: https://dietpi.com/phpbb/viewtopic.php?f=11&t=5502&p=16560#p16560
- Debian Jessie: libc6 v2.19
- armhf package deps: libc6 >= v2.22 🤔

- Checking latest v4.0.1 package: The same!
- `arm64` package available: The same!
- => ARM-wide issue
- Even the oldest armhf package v3.2.50 requires v2.22, so armhf+Jessie did never work!
__________
Debian repo doesn't offer libc6 backports. Is it save to manually install the candidate from Stretch: https://packages.debian.org/stretch/libc6 ?
- No dependency issues, but it is **THE** basic library for everything, only safe if it is fully backwards compatible...

**Comment by MichaIng** [comment]
Okay, I am not keen to offer a `libc6` update. This is too basic/risky, since nearly everything relies on that library.
**EDIT: Just tested on Jessie VM, and locales depends on libc6 << 2.24 anyway:**
```
root@VM-Jessie:/tmp# dpkg -i libc6_2.24-11+deb9u3_amd64.deb
dpkg: regarding libc6_2.24-11+deb9u3_amd64.deb containing libc6:amd64:
 libc6:amd64 breaks locales (<< 2.24)
  locales (version 2.19-18+deb8u10) is present and installed.

dpkg: error processing archive libc6_2.24-11+deb9u3_amd64.deb (--install):
 installing libc6:amd64 would break locales, and
 deconfiguration is not permitted (--auto-deconfigure might help)
Errors were encountered while processing:
 libc6_2.24-11+deb9u3_amd64.deb
```
- And there is no v2.23 or v2.22 package available: http://deb.debian.org/debian/pool/main/g/glibc/
- Its `libc-bin`, AFAIK: https://packages.debian.org/de/jessie/libc-bin
- Rats tail... do never touch this...!
_________
Only solutions:
- Disable install option for ARM+Jessie
- Compile own packages for ARM+Jessie
  - @Fourdee 
Do you think it's still worth it?
_______
Other topic:
- Update emby-server to current v4.0.1 release, which offer native arm64 packages as well. PR up: https://github.com/Fourdee/DietPi/pull/2525

**Comment by MichaIng** [comment]
Okay for now I disabled Jessie + ARM support. I think the effort is simply too high to create own binaries/packages for the low amount of remaining Jessie users. And since this anyway never worked (see above), at least no user will miss it: https://github.com/Fourdee/DietPi/pull/2525/commits/885abcf886e136dc7d40ff55fcdf225c27347fc3

Will mark as closed.
